README.md

```diff
@@ -140,28 +140,39 @@ tox -e kubeflow-local
 ```
 
 ### Run behind proxy
+
 #### Prerequistes
 **To run the tests behind proxy using Notebook or using the driver, the following step is necessary:**
```

Comment:
with the new implementation, this will be a pre-requisite for running with Notebook only. let's make that clear.